loops.js

```diff
@@ -16,9 +18,16 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
-
-
+// ==========================================
+for (let i = 0; i < 5; i++) {
+    // i begint bij 0, dus 0, 1, 2, 3 geven "loop..." i < 4
+    if (i < 4) {
```

Comment:
Prima! wat ook kan is het volgende met triple = teken je checkt nu ook of het zelfde type is dus een nummer en niet een string "4" persoonlijk doe ik het liever op deze manier maar die van jou werkt ook prima hoor 👍🏼 
```suggestion
    if (i === 4) {
```